ignite/pkg/xgenny/runner.go

```diff
@@ -0,0 +1,140 @@
+package xgenny
+
+import (
+	"context"
+	"io"
+	"os"
+	"path/filepath"
+
+	"github.com/gobuffalo/genny/v2"
+
+	"github.com/ignite/cli/v28/ignite/pkg/placeholder"
+	"github.com/ignite/cli/v28/ignite/pkg/randstr"
+	"github.com/ignite/cli/v28/ignite/pkg/xos"
+)
+
+type Runner struct {
+	*genny.Runner
+	Path     string
+	ctx      context.Context
+	tracer   *placeholder.Tracer
+	TempPath string
+}
+
+// NewRunner is a xgenny Runner with a logger.
+func NewRunner(ctx context.Context, appPath string) *Runner {
+	var (
+		runner  = genny.WetRunner(ctx)
+		tmpPath = filepath.Join(os.TempDir(), randstr.Runes(5))
+	)
+	r := &Runner{
+		ctx:      ctx,
+		Runner:   runner,
+		Path:     appPath,
+		TempPath: tmpPath,
```

Comment:
Is there a reason to keep these properties public? Otherwise if they only need to be read-only it might be better to make them private and only readable though getter methods

Reply:
not anymore, I will remove it